contents/docs/tutorials/toolbar.md

```diff
@@ -0,0 +1,185 @@
+---
+title: Visualizing User Behavior - Toolbar
+sidebar: Docs
+showTitle: true
+---
+<br />
+
+<small class="note-block centered">_Estimated Reading Time: 6 minutes ☕☕_</small>
+
+<br />
+
+<span class="larger-image">
+
+![Toolbar banner image](../../images/tutorials/banners/toolbar.png)
+
+</span>
+
+<br />
+
+An underlying purpose of analytics is to answer questions by utilizing data.
+
+_"Who's using my website?"_ 
+
+_"What type of people pay for my product?"_
+
+_"Why am I not retaining users?"_
+
+Often times, these questions don't have a simple answer, and, more often than not, just getting _any_ answer to your questions requires a good amount of effort. 
+
+However, there are some questions that have straightforward answers - the difficulty comes in getting that answer across. 
+
+Enter the PostHog Toolbar. 
+
+Consider the question: _"Where are my users clicking?"_
+
+![Toolbar landing image](../../images/tutorials/toolbar/toolbar-landing-page.png)
+
+How's that for an answer?
+
+This tutorial will walk you through using the PostHog Toolbar to answer some of your analytics questions, as well as show you step-by-step how to create actions with it, so that you can sort through your events and level up your PostHog game. 
+
+
+<span class='centered'>...</span>
+
+_Prefer to watch a video? Check out the [Toolbar sections of our demo video](https://youtu.be/aUILrrrlu50?t=48) for a shorter overview of this functionality._
+
+## Pre-Requisites
+
+To follow this tutorial along, you need to:
+
+1. Have [deployed PostHog](/docs/deployment).
+1. Have added the [PostHog snippet](/docs/integrations/js-integration) to your website. Our Toolbar only works with our [JavaScript Integration](/docs/integrations/js-integration), and requires the PostHog snippet to work. A Segment snippet, for example, will not give you access to the toolbar.
+
+## Why Use the Toolbar
+
+Before we go right into how to use the PostHog Toolbar, it's useful to understand why you'd want to do so.
+
+Our toolbar is still in Beta, and there are exciting new features for it on our roadmap. However, in it's current form, it can:
+
+1. Display user clicks on your website in a visual way
+1. Tell you what element is the most clicked on a given page
+1. Create actions in a simple way
+1. Display all actions you have on a page (based on frontend elements)
+
+Our team uses the toolbar daily, mostly to get a quick overview of clicks (and therefore decide what pages need attention) as well as create actions. 
```

Comment:
Maybe add:

This helps us to do things like prioritize which parts of our website or application to redesign as any PostHog user can easily see the potential impact of their work. It also gives valuable context for how users are navigating your website or application, to help design the menus, information and product structure in a way that steers users towards the actions you would like them to take.